lib/jekyll/commands/clean.rb

```diff
@@ -4,8 +4,9 @@ class Clean < Command
       class << self
         def init_with_program(prog)
           prog.command(:clean) do |c|
-            c.syntax 'clean [subcommand]'
-            c.description 'Clean the site (removes site output and metadata file) without building.'
+            c.syntax "clean [subcommand]"
+            c.description "Clean the site
+            (removes site output and metadata file) without building."
```

Comment:
This will output literally and it's hard for static analysis to catch this.  Try:
```ruby
"string1 " \
  "string2"
```